octomap_server/launch/octomap_tracking_server.launch

```diff
@@ -1,6 +1,10 @@
 <launch>
+
+  <arg name="path" default=""/>
+  <arg name="changeIdFrame" default="/talker/changes"/>
```

Comment:
I don't think it's necessary to have the arg in addition to the param. One way to set it is more clear.